lib/matplotlib/ticker.py

```diff
@@ -242,6 +245,9 @@ def set_view_interval(self, vmin, vmax):
     def get_minpos(self):
         return self._minpos
 
+    def set_minpos(self, minpos=0):
```

Comment:
*shrugs* then I'll just write it off as out of scope for this PR and a debatable major overhaul for like a version 3.0